polkapox/1.0.0-beta/Dockerfile

```diff
@@ -0,0 +1,85 @@
+##### Building stage #####
+
+# Stage 1: Build Python environment with required packages
+FROM python:3.8-slim AS app
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1
+
+# Add metadata
+LABEL authors="CDC PolkaPox Team"
+LABEL description="Container used in the PolkaPox pipeline by graph_reconstruct.nf module"
+LABEL base.image="python:3.8-slim"
+LABEL dockerfile.version="1.0"
+LABEL website="https://github.com/CDCgov/polkapox"
+LABEL license="https://github.com/CDCgov/polkapox/blob/master/LICENSE"
```

Comment:
We have a list of required labels for dockerfiles for this repo. It looks like you are missing some.

They are:
[X] base.image
[X] dockerfile.version
[ ] software (which I think would be polkapox)
[ ] software.version (which I think would be 1.0.0-beta)
[X] description
[X] website
[X] license
[ ] maintainer (which you can list as me if you don't want to list yourself)
[ ] maintainer.email

Reply:
added - thank you and I apologize for the oversight!